server/src/models.js

```diff
@@ -156,10 +158,22 @@ exports.shot = function shot(state) {
     [modelMap.get(key), metaMap.get(key)]
   ).then(
     ([data, meta]) => {
-      if (! (data && meta)) {
+      if (! data) {
         return Promise.reject(new Error("No data or returned from model"));
       }
-      return Promise.resolve({data: JSON.parse(data), meta: JSON.parse(meta), identifier: key});
+
+      // FIXME we need some way to configure the url
```

Comment:
I guess an environmental variable if it's too awkward to pull it out of the request.